release-notes/8.0/8.0.0/api-diff/Microsoft.NETCore.App/8.0.0_System.md

```diff
@@ -0,0 +1,841 @@
+# System
```

Comment:
System LGTM

This is namely obsoleting binary-formatter APIs, but also various other additions such as implementing the new generic math APIs (utf8 parse/format, lerp, deg2rad, rad2deg, etc), addition of more throw helpers on `ArgumentOutOfRangeException` (`ThrowIfZero`, `ThrowIfNegative`, `ThrowIfEqual`, `ThrowIfGreaterThan`, etc), a number of new `Span`/`ReadOnlySpan` APIs for searching or finding data (`ContainsAny`, `ContainsAnyExcept`, `CointainsAnyInRange`, `Count`, `IndexOfAny`, `IndexOfAnyExcept`, `LastIndexOf`, `Split`, etc), and new `Random` APIs in the form of `GetItems` and `Shuffle`

CC. @dotnet/area-system-runtime